eos/form-factors/parametric-bfw2010-impl.hh

```diff
@@ -224,6 +222,38 @@ namespace eos
         return a_T2_0 / (polynomials_T2[0] * x_T1);
     }
 
+    template <typename Process_>
+    double
+    BFW2010FormFactors<Process_, PToV>::_a_A1_0() const
```

Comment:
Works. Personally, I would have replaced a_A12_0, but that's not a reason to reject this!

Reply:
a_A12_0 is already replaced by the relation at q2 = 0 ;)